cli/lib/kontena/cli/certificate_command.rb

```diff
@@ -1,11 +1,13 @@
 
 class Kontena::Cli::CertificateCommand < Kontena::Command
 
-
+  subcommand ["list", "ls"], "List certificates", load_subcommand('certificate/list_command')
+  subcommand "show", "Show certificate details", load_subcommand('certificate/show_command')
   subcommand "register", "Register to LetsEncrypt", load_subcommand('certificate/register_command')
   subcommand "authorize", "Create DNS authorization for domain", load_subcommand('certificate/authorize_command')
   subcommand "get", "Get certificate for domain", load_subcommand('certificate/get_command')
```

Comment:
How many :crab: would I need to sacrifice to get this renamed to `kontena certificate request`?

We're changing the CLI anyways... need to also update the docs, and something for the changelog to explain how to move from the current secrets stuff to the new `certificates`?

Reply:
We could do that. And that way we could still use `kontena certificate get` for the old way of handling certs as secrets, with a deprecation warning of course. WDYT?

Reply:
I think that might be a good idea... with the domain authz changes it doesn't matter because the command still behaves the same for the user, but this removes all the `kontena certificate get` options (`--secret-name` `--cert-type` ) and also changes the way the resulting certificates are used... it might as well be a completely new command because it has very little in common with the old command.

If we're keeping the old API and mutation, then it wouldn't cost us anything to hang on to and just deprecate the old `kontena certificate get` command... and users might still have a legit need for managing certs using old secrets-based certificates, because support for those is hardcoded into the various LB stacks. It's not nice to force users to rewrite their stacks immediately across an upgrade by breaking the existing stacks, without giving them a chance to migrate gradually by being able to use both the old and new syntax?

Reply:
BTW it might also make it easier for users to upgrade if we picked up any existing `LE_CERTIFICATE_*` secrets in a migration, and created `Certificate` models for them?

Reply:
Added `request` command and made `get` use the old API.